docs/operator-ci-yaml.md

```diff
@@ -0,0 +1,58 @@
+# Operator Publishing / Review settings
+
+Each operator might have `ci.yaml` configuration file to be present in an operator directory (for example `community-operators/aqua/ci.yaml`). This configuration file is used by [community-operators](https://github.com/operator-framework/community-operators) pipeline to setup various features like `reviewers` or `operator versioning`.
```

Comment:
The path is not accurate (`community-operators/aqua/ci.yaml`). Please replace it with a valid path to an operator (`operators/aqua/ci.yaml`).